app/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-alpine
+
+RUN apk update
+ARG NODE_ENV=production
+ENV NODE_ENV=${NODE_ENV}
+
```

Comment:
By default the `node` image uses the root user. Consider switching to the `node` user at this point
```suggestion

# switch to unprivileged user from node base image
USER node

```
See
- https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#non-root-user
- https://github.com/nodejs/docker-node/issues/1169